client/sourceeditor/editor.js

```diff
@@ -46,10 +46,6 @@ const { OS } = Services.appinfo;
 // order to initialize a CodeMirror instance.
 
 const CM_STYLES   = [
-  "/devtools/client/themes/common.css",
-  "/devtools/client/sourceeditor/codemirror/lib/codemirror.css",
-  "/devtools/client/sourceeditor/codemirror/addon/dialog/dialog.css",
-  "/devtools/client/sourceeditor/codemirror/mozilla.css"
 ];
 
 const CM_SCRIPTS  = [
```

Comment:
Could all of these CM_SCRIPTS move to the HTML page instead (as script tags), and all the baseURI / scriptLoader stuff in this file can be removed?  I don't care so much about refactoring it as much as I don't think concatenating baseURI with each of these scripts will come up with the right path, and it's easier to come up with the right path from the HTML page